### PR TITLE
delete useless field messageLayer

### PR DESCRIPTION
**Pull request by kadisi**
Signed-off-by: zhangjie <iamkadisi@163.com>

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>

> /kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:
`messageLayer `field  is useless in `Controller struct`, we only use `upstream`  not Controller.messageLayer

### CODE REVIEW

File: cloud/pkg/controller/controller.go [hunk 1]
```diff
@@ -9,12 +9,10 @@ import (
 	"github.com/kubeedge/kubeedge/cloud/pkg/controller/config"
 	"github.com/kubeedge/kubeedge/cloud/pkg/controller/constants"
 	"github.com/kubeedge/kubeedge/cloud/pkg/controller/controller"
-	"github.com/kubeedge/kubeedge/cloud/pkg/controller/messagelayer"
 )
 
 // Controller use beehive context message layer
 type Controller struct {
```
Comment (sids-b) [inline cloud/pkg/controller/controller.go]: Controller will be empty structure now. Can you open and close the curly brackets at the same line please ?

Comment (kadisi) [reply cloud/pkg/controller/controller.go]: ok

**Comment by rohitsardesai83** [comment]
@kadisi ,please run gofmt.

**Pull request merged**